### PR TITLE
chore: add go mod

### PR DESCRIPTION
**Pull request by Decodetalkers**
go mod add

### CODE REVIEW

**Comment by wineee** [comment]
https://build.deepin.com/package/show/home:rewine:gomod_test/deepin-network-proxy
deb 可以构建成功
cc @black-desk

**Comment by black-desk** [comment]
> https://build.deepin.com/package/show/home:rewine:gomod_test/deepin-network-proxy deb 可以构建成功 cc @black-desk

意思是说不用管这个ci了么

File: Makefile [hunk 1]
```diff
@@ -17,13 +18,15 @@ export GO111MODULE=off
 all: build
 
 prepare:
+	@mkdir -p ${GODEP_DIR}/src/github.com/godbus/dbus/v5
+	@cp -r /usr/share/gocode/src/github.com/godbus/dbus/* ${GODEP_DIR}/src/github.com/godbus/dbus/v5
```
Comment (black-desk) [inline Makefile:22]: 你确定要搞得这么脏么

**Comment by wineee** [comment]
Superseded by https://github.com/linuxdeepin/deepin-network-proxy/pull/17

**Pull request closed without merging**